README.rst

```diff
@@ -48,6 +48,10 @@ use autoreconf::
    $ autoreconf -i
 
 
+and the dependency for this is build-essential. There is also support for
```

Comment:
I don't think we should refer to 'build-essential' since that is distro specific.  I think just telling people to run autoreconf is enough.

As for the cmake comment I feel that it should not be mentioned without some basic usage information (at least a list of commands to perform a default build).  No need for this document to state that it was added in 2012, we should be concise / not include information that does not directly help someone understand how to build the library.  If someone is interested in historic details they can look at the git history.